implementations/micrometer-registry-dynatrace/src/main/java/io/micrometer/dynatrace/v2/DynatraceExporterV2.java

```diff
@@ -209,11 +222,11 @@ private String createGaugeLine(Meter meter, Map<String, String> seenMetadata, Me
                         meter.getId().getName()));
                 return null;
             }
-            Metric.Builder metricBuilder = createMetricBuilder(meter).setDoubleGaugeValue(value);
+            MetricLineBuilder.GaugeStep metricBuilder = createMetricBuilder(meter).gauge();
```

Comment:
I guess there is some naming issue here: `createMetricBuilder` does not create a `Metric.Builder` and `metricBuilder` seems to be a `GaugeStep`.

I think this should be:
```java
MetricLineBuilder.GaugeStep gaugeStep = createTypeStep(meter).gauge();
```

Reply:
Yes that's clearer. Applied https://github.com/micrometer-metrics/micrometer/commit/98f35145d1a7ddb07661c85cf00652f1e0dcb456.